### PR TITLE
OSDOCS-6479: update RHDE page with compat matrix

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s): 
4.0 RHDE
In re: MicroShift 4.14/RHEL 9.2

Issue:
https://issues.redhat.com/browse/OSDOCS-6479

Link to docs preview:
https://63486--docspreview.netlify.app/openshift-rhde/latest/overview/rhde-overview

QE review:
- N/A

Additional info: 
OK to merge now; page will not go live until GA.

### CODE REVIEW

File: modules/about-rhde.adoc [hunk 1]
```diff
@@ -32,19 +34,31 @@ The two products of {product-title} work together as a single solution for devic
...
 
 |8.7
 |4.12
 |Developer Preview
+|No
 
 |9.2
 |4.13
 |Technology Preview
+|4.13.z
```
Comment (dhellmann) [inline modules/about-rhde.adoc:47]: We do not support upgrading 4.13.

File: modules/about-rhde.adoc [hunk 2]
```diff
@@ -6,45 +6,58 @@
...
 |*{microshift} release status*
+|*{microshift} supported updates*
 
-|8.7
-|4.12
-|Developer Preview
+|9.2
+|4.14
+|Generally Available
+|4.14.0-4.14.z and 4.14-4.15
```
Comment (DanielFroehlich) [inline modules/about-rhde.adoc:42]: can add and "arrow" to make clear FROM -> TO we do support? i.e. 4.14.0 -> 4.14.z, 4.14.z -> 4.15.z

Comment (DanielFroehlich) [reply modules/about-rhde.adoc:42]: otherwise, looks good to me!

File: modules/about-rhde.adoc [hunk 3]
```diff
@@ -6,45 +6,61 @@
 [id="about-rhde_{context}"]
 = About {product-title}
 
-{product-title} is for organizations who need small-form-factor edge devices with support for bare metal, virtualized, or containerized applications. {product-title} helps address many of the emerging questions around large-scale edge computing at the device edge by incorporating Kubernetes features in a new, smaller, lighter-weight footprint with an edge-optimized Linux OS built from the world's leading enterprise Linux platform in {op-system}.
+{product-title} is for organizations who need small-form-factor edge devices and support for bare metal, virtualized, or containerized applications. {product-title} helps address many of the emerging questions around large-scale edge computing at the device edge by incorporating Kubernetes features in a new, smaller, lighter-weight footprint with an edge-optimized Linux OS built from the world's leading enterprise Linux platform in {op-system}.
```
Comment (gaurav-nelson) [inline modules/about-rhde.adoc:9]: **nitpick:** Use 'operating system' instead of 'OS'.

Comment (gaurav-nelson) [inline modules/about-rhde.adoc:9]: **suggestion:** "who need" > "that"

File: overview/rhde-overview.adoc [hunk 4]
```diff
@@ -3,9 +3,6 @@
 include::_attributes/common-attributes.adoc[]
 :context: device-edge-overview
 
-{product-title} delivers an enterprise-ready distribution of the Red Hat-led open source community project {microshift}, a lightweight Kubernetes orchestration solution built from the edge capabilities of Red Hat OpenShift, along with an edge-optimized operating system built from {op-system-first}.
-
-:FeatureName: {product-title}
-include::snippets/technology-preview.adoc[]
+{product-title} delivers an enterprise-ready distribution of the Red Hat-led open source community project {microshift-first}. {microshift} is a lightweight Kubernetes orchestration solution built from the edge capabilities of {OCP}, along with an edge-optimized operating system built from {op-system-first}.
```
Comment (gaurav-nelson) [inline overview/rhde-overview.adoc:6]: **nitpick:** s/open source/open-source

**Comment by gaurav-nelson** [comment]
**praise:** 👏 Good work. 

I've marked a few nits; we can merge this after the fixes.

**Comment by mburke5678** [comment]
@ShaunaDiaz Do we want to wait on the merge until all of the linked docs are live? (I get a few 404s; I am assuming those docs are in process.) That way, we can make sure the links work?

**Comment by ShaunaDiaz** [comment]
@mburke5678 I wasn't thinking. Yes, this needs to wait until OCP release. Thank you for the reminder.

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**